pandas/tests/reshape/test_get_dummies.py

```diff
@@ -169,7 +177,7 @@ def test_get_dummies_unicode(self, sparse):
         e = "e"
         eacute = unicodedata.lookup("LATIN SMALL LETTER E WITH ACUTE")
         s = [e, eacute, eacute]
-        res = get_dummies(s, prefix="letter", sparse=sparse)
+        res = get_dummies(s, dtype=np.uint8, prefix="letter", sparse=sparse)
```

Comment:
Wouldn't we rather just catch the warnings for these? Wondering how we remember in the future to go back and update these tests when we make the change to the dtype

Reply:
Could do, I just thought that would be a lot of warnings to add

Regarding updating tests - I wouldn't have thought they needed updating, I'd have thought just having a test which called `.get_dummies()` (without specifying `dtype`) would be enough

Reply:
Yea I agree. So that's why I was thinking it is better to catch the warning for now and not change the argument. Otherwise with this in the future we lose testing the behavior of the default argument unless someone comes back and revert what was changed here

Reply:
no all tests should be fixed now
and then u have an explicit test of the warning

it's not better to defer fixing something like this

Reply:
OK good points, thanks for raising

I've added this to the agenda for the next dev meeting

@kianelbo let's hold off further changes til after there's been discussion